posthog/migrations/max_migration.txt

```diff
@@ -1 +1 @@
-0869_create_cohort_calculation_history
\ No newline at end of file
+0869_create_cohort_calculation_history
```

Comment:
Shouldn't this be one higher?

Reply:
oh, should it... damn, i manually recreated 🙈

Reply:
fixed 💖